worlds/pokemon_rb/regions.py

```diff
@@ -1944,7 +1945,8 @@ def create_regions(self):
                     connect(multiworld, player, region.name, entrance_data["to"]["map"],
                             lambda state: logic.rock_tunnel(state, player), one_way=True)
                 else:
-                    connect(multiworld, player, region.name, entrance_data["to"]["map"], one_way=True)
+                    connect(multiworld, player, region.name, entrance_data["to"]["map"], one_way=True,
+                            name=entrance_data["name"] if "name" in entrance_data else None)
```

Comment:
```suggestion
                            name=entrance_data.get("name", None))
```

Should be cleaner in the future.